src/components/json/worldData.js

```diff
@@ -9,6 +9,7 @@ import karthik from "../../photos/karthik.jpg";
 import MuhammadHassanRazaa from "../../photos/MuhammadHassanRazaa.jpg";
 import advaithunnikrishnan from "../../photos/advaithunnikrishnan.jpg";
 import AsishRaju from "../../photos/AsishRaju.jpg";
+import Aineshphanithi from "../../photos/Ainesh.jpg";
```

Comment:
@Aineshphanithi 
This should be
import Aineshphanithi from "../../photos/Aineshphanithi.jpg";

Reply:
Hey, just make this change on the main branch and you'll be gtg.

Reply:
```suggestion
import Aineshphanithi from "../../photos/Aineshphanithi.jpg";
```